modules/core/src/v2/coins/algo.ts

```diff
@@ -257,8 +257,14 @@ export class Algo extends BaseCoin {
 
         const isTokenTx = this.isTokenTx(txJson.type);
         if (isTokenTx) {
+          const type = accountLib.Algo.algoUtils.getTokenTxType(
```

Comment:
Changes looks good, please create a generic version and test with the platform integration.